Library/Formula/python3.rb

```diff
@@ -77,8 +80,6 @@ def install
       --with-system-ffi
     ]
 
-    args << "--without-gcc" if ENV.compiler == :clang
```

Comment:
Hm, is this no longer needed?

Reply:
Doesn't appear to be.
Saw this fly by during a build:
`configure: WARNING: unrecognized options: --without-gcc`

Reply:
OK, yup - looks like it went away in python 3.9.0: https://bugs.python.org/issue33483